src/lib.rs

```diff
@@ -142,7 +142,9 @@ impl Wal {
             dir
         };
 
-        dir.try_lock_exclusive()?;
```

Comment:
for next time I'd be good if you could explain the non trivial changes in the PR description

I checked the changelog myself https://github.com/al8n/fs4-rs/blob/main/CHANGELOG.md#0130